app/controllers/sightings_controller.rb

```diff
@@ -46,7 +46,7 @@ def destroy
     private
 
     def sighting_params
-            params.require(:sighting).permit(:animal_id, :sighting_date)
+            params.require(:sighting).permit(:start_date, :end_date, :animal_id, :sighting_date)
```

Comment:
It looks like your schema has location as a column which most likely should be included in here as well.